src/pages/components/progress-indicator/style.mdx

```diff
@@ -41,12 +51,12 @@ outlined circle.
 
 <div className="image--fixed">
 
-![Examples of current, completed, and future steps for progress indicator](images/progress-indicator-style-1.png)
+![Examples of current, completed, and future steps for progress indicator](images/progress-indicator-style-2.png)
```

Comment:
You need to use the other image component for this one.

<img width="896" alt="image" src="https://user-images.githubusercontent.com/11670886/173896498-d9f0d5db-43c3-452c-9f6a-6f47eacd2fed.png">